<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FortEdit</class>
 <widget class="QMainWindow" name="FortEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FortEdit</string>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="MapNameLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="MapNameLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="placeholderText">
         <string>Untitled</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="DisplayModeLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Display Mode:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="DisplayModeCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentText">
         <string>2D</string>
        </property>
        <item>
         <property name="text">
          <string>2D</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3D</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Actual 3D</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="PlayButton">
        <property name="toolTip">
         <string>Invoke the game engine with current map</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/tool-icons/res/play-button.png</normaloff>:/tool-icons/res/play-button.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="GridCheckBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Grid</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ObjectVisibilityToggle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Object Visibility</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="WidthLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Width</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="MapWidthSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="HeightLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="MapHeightSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>8</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="ToolBox">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="ToolSelect">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Select tool</string>
          </property>
          <property name="text">
           <string>Select Tool</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/tool-icons/res/tool-select.png</normaloff>:/tool-icons/res/tool-select.png</iconset>
          </property>
          <property name="shortcut">
           <string>V</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonIconOnly</enum>
          </property>
          <property name="autoRaise">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="ToolObject">
          <property name="toolTip">
           <string>Place castle tool</string>
          </property>
          <property name="text">
           <string>Castle Tool</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/tool-icons/res/tool-object.png</normaloff>:/tool-icons/res/tool-object.png</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonIconOnly</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="ToolTerritory">
          <property name="toolTip">
           <string>Territory placement tool</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/tool-icons/res/tool-territory.png</normaloff>:/tool-icons/res/tool-territory.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="ToolTerrain">
          <property name="toolTip">
           <string>Place accessible terrain tool</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/tool-icons/res/tool-terrain.png</normaloff>:/tool-icons/res/tool-terrain.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="AudioOptions">
          <property name="toolTip">
           <string>Choose What Sounds the Map will Use</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/tool-icons/res/tool-audio.png</normaloff>:/tool-icons/res/tool-audio.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="LuaEditorButton">
          <property name="toolTip">
           <string>Edit AI Script</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/button-icons/res/lua-button.png</normaloff>:/button-icons/res/lua-button.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <widget class="QGraphicsView" name="GraphicsView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QTabWidget" name="EditorTabs">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="PropertiesTab">
         <attribute name="title">
          <string>Properties</string>
         </attribute>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::FieldsStayAtSizeHint</enum>
          </property>
          <item row="0" column="0">
           <widget class="QStackedWidget" name="PropertyPageStack">
            <widget class="QWidget" name="BlankPage">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>3</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2"/>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>915</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="MenuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="ActionNew"/>
    <addaction name="ActionOpen"/>
    <addaction name="ActionSave"/>
    <addaction name="ActionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="ActionSaveSoundSet"/>
    <addaction name="ActionLoadSoundSet"/>
   </widget>
   <addaction name="MenuFile"/>
  </widget>
  <action name="ActionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="toolTip">
    <string>Open a FortNitta map file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="ActionSaveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="toolTip">
    <string>Save the current map</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="ActionNew">
   <property name="text">
    <string>New map</string>
   </property>
   <property name="toolTip">
    <string>Create a new map</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="ActionSaveSoundSet">
   <property name="text">
    <string>Save Sound Set...</string>
   </property>
  </action>
  <action name="ActionLoadSoundSet">
   <property name="text">
    <string>Load Sound Set...</string>
   </property>
  </action>
  <action name="ActionSaveAiScript">
   <property name="text">
    <string>Save AI Script</string>
   </property>
  </action>
  <action name="ActionOpenAiScript">
   <property name="text">
    <string>Open AI Script</string>
   </property>
  </action>
  <action name="ActionSave">
   <property name="text">
    <string>Save...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
